### PR TITLE
Added ARO comment command

### PR DESCRIPTION
**Pull request by FieldEffect-OperationalDevelopment**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
- Added new command to comment on AROs

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by maimorag — CHANGES_REQUESTED** [review]
Hi @FieldEffect-OperationalDevelopment  ,
Thank you for your contribution!
Good work :)

File: Packs/CovalenceManagedSecurity/ReleaseNotes/1_3_0.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+
+##### Covalence Managed Security
+
+- Added support for commenting on an ARO.  
+- Added 5 commands:  
+  - ***cov-mgsec-comment-aro***
```
[the remark this answers is not in the document]
Comment (FieldEffect-OperationalDevelopment) [reply Packs/CovalenceManagedSecurity/ReleaseNotes/1_3_0.md]: Done, thank you.

**Comment by maimorag** [comment]
Hi @FieldEffect-OperationalDevelopment, the code looks good!

We're ready for a demo. Please check [this page](https://xsoar.pan.dev/docs/contributing/demo-prep), and let me know when you're available for one over DFIR. 
Feel free also to send me a recording of a demo.

**Comment by maimorag** [comment]
Hi @FieldEffect-OperationalDevelopment, we haven’t heard from you in a while.
Do you need any help with the pull request?


Please feel free to reach out to me here or on Slack.
Thanks again for contributing to our repo, hope to hear from you soon.

> Hi @FieldEffect-OperationalDevelopment, the code looks good!
> 
> We're ready for a demo. Please check [this page](https://xsoar.pan.dev/docs/contributing/demo-prep), and let me know when you're available for one over DFIR. Feel free also to send me a recording of a demo.

**Review by maimorag — APPROVED** [review]

**Pull request merged**